src/client/reducers/reducer_moviesFull.js

```diff
@@ -7,18 +7,21 @@ const initialCache = {
 
 export default function (state = initialCache, action) {
     switch (action.type) {
-        case C.PUT_MOVIE_TO_CACHE:
-            console.log('PUT_MOVIE_TO_CACHE', action.payload);
+        case C.PUT_MOVIE_TO_CACHE: {
+            // console.log('PUT_MOVIE_TO_CACHE', action.payload);
```

Comment:
@vitkuz please remove `console.log` if it's not needed.

Reply:
Done